### PR TITLE
Resposive Navbar created

### PR DESCRIPTION
**Pull request by Tushar98644**
Screenshot 2022-12-09 at 12 01 46 AM

Screenshot 2022-12-09 at 12 01 55 AM
Screenshot 2022-12-09 at 12 02 08 AM
Screenshot 2022-12-09 at 12 02 00 AM
Screenshot 2022-12-09 at 12 01 38 AM
Screenshot 2022-12-09 at 12 01 32 AM

### CODE REVIEW

**Comment by Tushar98644** [comment]
https://user-images.githubusercontent.com/107763774/206543808-d96586f2-5c61-4a39-a785-5db21ac24c6f.mp4

**Comment by Komachian** [comment]
Please **design the navbar as given below** 👇🏽. You'll find the required assets inside `public/navbar`.
Navbar

**Comment by Tushar98644** [comment]
Screenshot 2022-12-10 at 2 29 11 AM
updated

File: styles/nav.module.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+.navbar_2{
+  position: absolute;
+  left: 44%;
+}
+ .navbar_1{
+  width: 100%;
+  position: absolute;
+  top:25%;
+  transform: rotateX(180deg);
```
Comment (Komachian) [inline styles/nav.module.css]: Remove this line of code

File: styles/nav.module.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
+.nav_div{
+  position: relative;
+  background-color: rgb(34, 198, 201);
```
Comment (Komachian) [inline styles/nav.module.css]: Remove this line of code

**Review by Komachian — APPROVED** [review]
Good job!

**Pull request closed without merging**